src/Resources/public/contao_files/templates/smartgear/mod_newslist.html5

```diff
@@ -1,15 +1,10 @@
-<?php $this->extend('block_unsearchable'); ?>
-
 <?php $this->block('content'); ?>
-  <?php $this->insert('filters', ['filters'=>$this->filters ?? [],'config'=>$this->config ?? [],'dataContainer'=>'listNews']); ?>
```

Comment:
This line seems to add filters. It is removed because there is no filter template to add or is it a mistake?

Reply:
restored, my bad